openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2152,9 +2152,9 @@ The following is a non-normative example of an object containing the `credential
 The following additional claims are defined for authorization details of type `openid_credential` and this Credential Format.  
 
 * `claims`: OPTIONAL. An array of claims description objects as defined in (#claims-description-authorization-details).
-* `credential_definition`: OPTIONAL. Object containing the detailed description of the Credential. It consists of the following parameters:
-    * `@context`: OPTIONAL. Array as defined in (#server-metadata-ldp-vc). It MUST only be present if the `format` claim is present in the root of the authorization details object. It MUST not be present otherwise. 
-    * `type`: OPTIONAL. Array as defined in (#server-metadata-ldp-vc).  This claim contains the type values the Wallet requests authorization for at the Credential Issuer. MUST only be present if the `@context` claim is present. 
+* `credential_definition`: REQUIRED. Object containing the detailed description of the Credential. It consists of the following parameters:
+    * `@context`: REQUIRED. Array as defined in (#server-metadata-ldp-vc). It MUST only be present if the `format` claim is present in the root of the authorization details object. It MUST not be present otherwise. 
+    * `type`: REQUIRED. Array as defined in (#server-metadata-ldp-vc).  This claim contains the type values the Wallet requests authorization for at the Credential Issuer. MUST only be present if the `@context` claim is present. 
```

Comment:
```suggestion
    * `type`: REQUIRED if the `format` claim is present with a value of `ldp_vc`. It MUST NOT be present otherwise. Array as defined in (#server-metadata-ldp-vc).  This claim contains the type values the Wallet requests authorization for at the Credential Issuer.
```